<?xml version="1.0" ?>
<robot name="gown_box">
  <link name ="gown">
    <!--<inertial>
        <mass value="3.92"/>
        <origin
                xyz="0.0 0.07825 0.0"
                rpy="0 0 0" />
    </inertial>-->
    <visual name="visual">
      <origin
        xyz="0.0 0.0 -0.1"
        rpy="0 0 0" />
      <geometry>
        <box size="0.04 0.12 0.2"/>
      </geometry>
      <!--<color>1.0 0.5 0.0</color>-->
    </visual>
    <collision name="collision">
      <origin
        xyz="0.0 0.0 -0.1"
        rpy="0 0 0" />
      <geometry>
        <box size="0.04 0.12 0.2"/>
      </geometry>
    </collision>
  </link>
</robot>
